src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -9,8 +9,7 @@ import Toolbar from '@material-ui/core/Toolbar';
 
 import Version from '../../../../../package.json';
 import './Banner.css';
-
-const theme = createMuiTheme();
+import theme from '../../theme';
 
 const useStyles = makeStyles({
```

Comment:
Instead of importing a theme, here, could we instead alter the `makeStyles` call to inherit a theme ([like the `myfeeds` component does](https://github.com/Seneca-CDOT/telescope/blob/a38207c1deee36f69deb2972a16facef6dcadfa2/src/frontend/src/pages/myfeeds.js#L7))?

https://github.com/Seneca-CDOT/telescope/blob/a38207c1deee36f69deb2972a16facef6dcadfa2/src/frontend/src/pages/myfeeds.js#L7

Reply:
Seems a bit of a cleaner solution if we can pull it off. Thoughts, @humphd?

Reply:
Does `const useStyles = makeStyles(theme => ({ ...` create a new theme? I'd like to use the `theme.js` file. All our components should share a theme.

Reply:
As far as I understand, this syntax allows our default theme (the one defined in `theme.js` and [_provided_ in TopLayout.js](https://github.com/Seneca-CDOT/telescope/blob/7128ecfae2a77ac31605864dfed3c4afa5cd8abd/src/frontend/plugins/gatsby-plugin-top-layout/TopLayout.js#L5-L18)) to be passed as an argument to `makeStyles`.